platform/platform-filter-delegate/pom.xml

```diff
@@ -77,7 +77,7 @@
                 <configuration>
                     <instructions>
                         <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
-                        <Embed-Dependency>platform-util</Embed-Dependency>
+                        <Embed-Dependency>platform-util,guava</Embed-Dependency>
```

Comment:
i was able to get to work by adding guava to the features.xml file that load platform-filter-delegate